package.json

```diff
@@ -29,12 +29,11 @@
     "chai": "^4.3.4",
     "coveralls": "^3.1.1",
     "eslint": "^8.5.0",
-    "eslint-config-standard": "^16.0.3",
+    "eslint-config-standard": "^17.0.0",
     "eslint-plugin-chai-friendly": "^0.7.2",
     "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-node": "^11.1.0",
```

Comment:
Can we do this PR without these updates?

Reply:
We can, but the "npm install" command won't work when you clone the repo and try to build the library.

Reply:
I understand. Have you tried running `npm install` without having these two lines edited locally (package.json file)?

Reply:
Yes, the first time I cloned the repo when I tried to run "npm install" it didn't work, I had to resolve the package conflicts first. If this is a mistake from my side, please let me know.

Reply:
Yeah, I believe it's something on your end, since I just tried cloning the repo from scratch and running `npm it` and it worked flawlessly. The lock file was left intact.
So let's remove these package changes from this PR.